oap-stdlib/src/main/java/oap/message/Message.java

```diff
@@ -34,15 +34,17 @@
 final class Message {
```

Comment:
It will be very useful, if you place javadoc for `Message`. It will help to understand developers to understand this object

Reply:
Persistent asynchronous transactional binary protocol for sending data, for instance:
- OAP statsDB uses it (Sender and Listener)
- ....
please specify other examples